src/platform/imx8ulp/include/platform/lib/dma.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Peng Zhang <peng.zhang_8@nxp.com>
+ */
+
+#ifdef __SOF_LIB_DMA_H__
+
+#ifndef __PLATFORM_LIB_DMA_H__
+#define __PLATFORM_LIB_DMA_H__
+
+#define PLATFORM_NUM_DMACS	3
+
```

Comment:
Is this really 3? We have EDMA and Dummy DMA. Am I missing something?

Reply:
It should be 2. Will update.